src/element/element_writer.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright Amazon.com, Inc. or its affiliates.
+
+//! Provides utility to serialize Ion data from [`Element`] into common targets
+//! such as byte buffers or files.
+
+use crate::result::IonResult;
+use crate::{Element, Value};
+
+/// Serializes [`Element`] instances into some kind of output sink.
+pub trait ElementWriter {
```

Comment:
🗺️ This trait used to live in the same module as the now-removed `IonWriter` trait. It has been moved to its own file/module.